HacktoberfestProject.Web/Startup.cs

```diff
@@ -64,12 +68,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 			app.UseAuthentication();
 			app.UseAuthorization();
 
-      app.Use(async (ctx, next) =>
-      {
-          ctx.Response.Headers.Add("Content-Security-Policy", "default-src 'self' 'unsafe-inline'; connect-src 'self' https://api.github.com;frame-src *.youtube.com; img-src 'self' *.githubusercontent.com; style-src-elem 'self' *.bootstrapcdn.com;");
-          await next();
-      });
-      
+	  app.Use(async (ctx, next) =>
+	  {
+		  ctx.Response.Headers.Add("Content-Security-Policy", "default-src 'self' 'unsafe-inline'; connect-src 'self' https://api.github.com;frame-src *.youtube.com; img-src 'self' *.githubusercontent.com; style-src-elem 'self' *.bootstrapcdn.com;");
```

Comment:
We need to make sure that we don't get `*.bootstrapcdn.com` back in the CSP when we merge this 🤓